client-js/app/elements/labrad-plot.ts

```diff
@@ -222,6 +247,55 @@ export class Plot extends polymer.Base {
             .attr('y', 0)
             .attr('width', width)
             .attr('height', height);
+
+
+    // Color Bar Axis
+    if (this.numIndeps == 2) {
+      p.zAxis = d3.svg.axis()
+              .scale(p.zScale)
+              .orient('right')
+              .ticks(7)
+              .tickSize(5);
+
+      // The Color Bar Gradient
+      p.svg.append('defs').append("linearGradient")
+          .attr("id", "ColorBarGradient")
+          .attr("x1", "0%")
+          .attr("y1", "0%")
+          .attr("x2", "0%")
+          .attr("y2", "100%")
+          .selectAll("stop")
+          .data([
+            {offset: "0%", color: COLOR_MAP[0]},
+            {offset: "50%", color: COLOR_MAP[128]},
+            {offset: "100%", color: COLOR_MAP[255]}
+          ])
```

Comment:
Is this relying on d3 to do the interpolation between `COLOR_MAP` values? That's probably ok, though the matplotlib guys who spent all that time carefully designing the viridis map would probably be sad. It might be nice to use a few more points, just to make improve the fidelity of the gradient and the actual color map (note that this gradient data can be precomputed once when the color map data is first loaded).